### PR TITLE
Setting the project name on the core project and adding a plugin for …

### PR DESCRIPTION
**Pull request by gpoirier**
…bintray with a license (for bintray)

### CODE REVIEW

**Comment by jonas** [comment]
+1

File: project/plugins.sbt [hunk 1]
```diff
@@ -3,3 +3,4 @@
 addSbtPlugin("com.eed3si9n"      % "sbt-buildinfo"    % "0.4.0")
 addSbtPlugin("com.typesafe.sbt"  % "sbt-scalariform"  % "1.3.0")
 addSbtPlugin("com.typesafe.sbt"  % "sbt-git"          % "0.8.4")
+addSbtPlugin("me.lessis"         % "bintray-sbt"      % "0.3.0")
```
Comment (jonas) [inline project/plugins.sbt]: Is this going to be used in a future commit in this PR?

Comment (gpoirier) [inline project/plugins.sbt]: Yes, currently it's already possible to publish to a bintray account with this. But publishing to own account isn't that useful I suppose. What's missing is to have a ypg/mediative account as an organization, grant the right to the organization to people's account and then define the organization on the build file. At least that's my understanding, I haven't done it before.

Comment (jonas) [inline project/plugins.sbt]: Ok, I'll look into this. Could be nice to have Travis publish snapshot/releases based on tags as part of the build. I'll create a ticket.

Comment (jonas) [inline project/plugins.sbt]: Done in #8

**Pull request merged**